airflow/models/serialized_dag.py

```diff
@@ -37,9 +37,12 @@
 from airflow.utils import timezone
 from airflow.utils.session import provide_session
 from airflow.utils.sqlalchemy import UtcDateTime
+from airflow.www.security import SimpleAirflowSecurityManager
 
 log = logging.getLogger(__name__)
 
+security_manager = SimpleAirflowSecurityManager()
```

Comment:
We should probably move this to `write_dag` too otherwise import anything from `airflow/models/serialized_dag.py` will just need an extra second needlessly

Reply:
Do you mean the import too? Creating the object itself is fast.

Reply:
yup both the import and the object